src/views/Team/index.view.tsx

```diff
@@ -1,8 +1,55 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
 import * as React from "react";
+import { ReactComponent as PictureFrame } from "images/TeamPage/frame.svg";
+import { ReactComponent as LinkedinLogo } from "images/TeamPage/linkedin.svg";
+import MLHBanner from "components/MLHBanner/index.view";
+import { members } from "./members";
 import "./Team.scss";
 
+// const description = [
+//   {
+//     description:
+//       "Meet our team of creators, thinkers, strategists, and believers, collaborating together to create something amazing. ",
+//   },
+// ];
+
 const TeamPageView: React.FC = () => {
-  return <div className="TeamPage">Hello from TeamPageView</div>;
+  return (
+    <>
+      <MLHBanner />
+      <div className="TeamPage">
+        <div className="TeamPage__cardsContainer">
+          {Object.entries(members).map(([memberName, { role, linkedin }]) => (
+            <div key={memberName} className="TeamPage__individualCardContainer">
+              <PictureFrame className="TeamPage__pictureFrame" />
+              <img
+                src={
+                  require(`images/TeamPage/team/${memberName.replace(
+                    /\s/g,
+                    ""
+                  )}.jpg`).default
+                }
+                className="TeamPage__profilePicture"
+                width="140"
+                height="143"
```

Comment:
Do we plan on scaling each card relative to the current screen size or just have them wrap accordingly?